src/Reveal.Sdk.Dom/Visualizations/RadialVisualization.cs

```diff
@@ -3,10 +3,24 @@
 
 namespace Reveal.Sdk.Dom.Visualizations
 {
+    /// <summary>
+    /// The radial visualization is used for taking data and rendering it as collection of data points wrapped around a circle (rather than stretching along a horizontal line).

```

Comment:
as a collection of data points.... missing the "a". can also drop the parenthesis